client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks/useStocktakeLineEdit.ts

```diff
@@ -25,7 +25,9 @@ export const useStocktakeLineEdit = (
 ): useStocktakeLineEditController => {
   const t = useTranslation('inventory');
   const { id } = useStocktake.document.fields('id');
-  const nextItem = useNextItem(item?.id);
+  const { items } = useStocktake.line.rows();
+  const filtereredItems = items.filter(item => item.item?.id === item?.id);
```

Comment:
Spelling...

Reply:
There can be multiple stocktage lines for a single item, right?

Reply:
this is ironic, I think. picking up a spelling error and making one in the following comment!
there can be multiple stocktake lines for an item yep.